shared/src/components/Footer/Footer.test.tsx

```diff
@@ -14,46 +14,28 @@ describe("Footer", () => {
   });
 
   it("renders", () => {
-    const wrapper = shallow(<Footer version="2.7.0" />);
-    expect(wrapper).toMatchSnapshot();
+    const { container } = render(<Footer version="2.7.0" />);
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
Can you remove this test? I have never found these to be very useful as we should be testing explicit features, and they always require attention when changing layout even if functionally the component still works as expected.

Reply:
Yeah, not a huge fan of snapshot testing either. Will remove.

Updated docs: https://docs.google.com/document/d/1ysVlPByzVY-NBUjGic1WupHojUl30P6cf3Xe-OVRRPg/edit#heading=h.48e8g53iq33

Reply:
Done.